flex-and-box/src/app/extensions/FlexPlayground.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Box,
+  Select,
+  Tile,
+  Text,
+  Link,
+  Flex,
+  hubspot,
+  ButtonRow,
+} from '@hubspot/ui-extensions';
+import {
+  Setting,
+  defaultSettings,
+  settingsOptions,
+} from './enums/flexPlayground';
+
+const FlexPlayground = () => {
+  const [settings, setSettings] = useState({ ...defaultSettings });
+
+  const handleSettingChange = (key: string, value: any) => {
+    setSettings({ ...settings, [key]: value });
+  };
+
+  const handleRestoreDefaultSettings = () => {
+    setSettings({ ...defaultSettings });
+  };
+
+  const settingsAreDefault = Object.keys(settings).some(
+    (key) => settings[key] !== defaultSettings[key],
+  );
+
+  return (
+    <Flex direction={'column'} gap="xl">
+      <Box>
+        <Flex direction={'column'} gap="lg">
+          <Text>
+            Use this card to explore and try out the{' '}
+            <Link
+              href={
+                'https://developers.hubspot.com/docs/platform/ui-extension-components#layout-flex'
+              }
+            >
+              {'Flex'} component
+            </Link>
+            .
+          </Text>
+
+          <Flex {...settings}>
```

Comment:
could we set props individually instead of spreading? It's going to be easier for folks to read